2011-10-22	Ralf Glauberman	<ralfglauberman@gmx.de>

		#360805 Don't compile ipxe with pie on hardened.
		* /tools/firmware/etherboot/patches/ipxe-nopie.patche		New patch
		* /tools/firmware/etherboot/patches/series				Add ipxe-nopie.patch

--- a/tools/firmware/etherboot/patches/ipxe-nopie.patch	1970-01-01 01:00:00.000000000 +0100
+++ b/tools/firmware/etherboot/patches/ipxe-nopie.patch	2011-03-27 17:45:13.929697782 +0200
@@ -0,0 +1,11 @@
+--- ipxe/src/Makefile~	2011-03-27 17:41:52.000000000 +0200
++++ ipxe/src/Makefile	2011-03-27 17:43:20.869446433 +0200
+@@ -4,7 +4,7 @@
+ #
+ 
+ CLEANUP		:=
+-CFLAGS		:=
++CFLAGS		:= -nopie
+ ASFLAGS		:=
+ LDFLAGS		:=
+ MAKEDEPS	:= Makefile
--- a/tools/firmware/etherboot/patches/series	2011-03-25 11:42:50.000000000 +0100
+++ b/tools/firmware/etherboot/patches/series	2011-03-27 17:45:45.140446216 +0200
@@ -1,4 +1,5 @@
 boot_prompt_option.patch
 build_fix_1.patch
 build_fix_2.patch
 build_fix_3.patch
+ipxe-nopie.patch

